mydumper.c

```diff
@@ -1963,7 +1965,13 @@ GList * get_chunks_for_table(MYSQL *conn, char *database, char *table, struct co
 	if (!field) goto cleanup;
 
 	/* Get minimum/maximum */
-	mysql_query(conn, query=g_strdup_printf("SELECT %s MIN(`%s`),MAX(`%s`) FROM `%s`.`%s`", select_hint, field, field, database, table));
+	GString* query_string = g_string_new(NULL);
+	g_string_printf(query_string, "SELECT %s MIN(`%s`),MAX(`%s`) FROM `%s`.`%s`", select_hint, field, field, database, table);
+	if (where_clause) {
+		g_string_append_printf(query_string, " WHERE %s", where_clause);
+	}
+	query = g_string_free(query_string, false);
```

Comment:
```suggestion
	query = g_string_free(query_string, FALSE);
```

Reply:
??? I can't understand why we need using "FLASE" in caps

Reply:
If we want to use `false`, we should include `<stdbool.h>`, https://stackoverflow.com/a/4767943/1115857
the error message is what I got from release CI

Reply:
oh, incomprehensible c lang, thank you for that